src/libs/ui/src/organisms/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,166 @@
+import AccordionItem from '@ui/molecules/AccordianItem';
+import { Checkbox } from '@ui/molecules/CheckBox/Checkbox';
+import { useState } from 'react';
+
+const filterData = {
```

Comment:
Please move this const data to utils

Reply:
Moved the data to FilterData file in utils folder.
![image](https://github.com/user-attachments/assets/cac43175-57de-4143-8f34-e9111c88b95c)